spec/basicConf.js

```diff
@@ -4,6 +4,8 @@ var env = require('./environment.js');
 exports.config = {
   seleniumAddress: env.seleniumAddress,
 
+  framework: 'jasmine2',
```

Comment:
Ah but changing the default would break people who rely on it. Never mind, don't do that.